sm_annotation_utils/sm_annotation_utils.py

```diff
@@ -248,13 +251,46 @@ def get_tables(self, datasets, sf_adduct_pairs, fields=['msm', 'fdr'], db_name='
 
     def metadata(self, datasets):
         """
-        Pandas dataframe where rows are flattened metadata JSON objects
+        Pandas dataframe for a subset of datasets where rows are flattened metadata JSON objects
         """
         from pandas.io.json import json_normalize
         df = json_normalize([d.metadata.json for d in datasets])
         df.index = [d.name for d in datasets]
         return df
 
+    def get_annotations(self, fdr=0.1, db_name='HMDB', timeout='240s'):
+        """
+        Returns a table of booleans indicating which ions were annotated in a particular dataset at the specified fdr
+        Known issue: only returns datasets with some anntotation at the given FDR level. Use in conjunction with get_metadata() to get a full dataset list
+        :param fdr: fdr level to export annotations at (should be one of [0.05, 0.1, 0.2, 0.5]
+        :param db_name: database to search against
+        :return: pandas dataframe index=dataset ids, columns(multindex) adduct / molecular formula
+        """
+        s = Search(using=self._es_client, index=self._es_index).params(timeout=timeout)\
+                    .filter('term', db_name=db_name)\
+                    .filter("range", **{'fdr': {'to': fdr}})\
+                    .fields(['sf', 'adduct', 'ds_id', ''])[:2147483647]
```

Comment:
it's @intsco's  'big number' (maybe should be replaced with sys.maxint) to get the execute command to return the maximum possible number of records